docs/UserGuide.md

```diff
@@ -181,6 +144,92 @@ Expected output (fail):
 Oops! Invalid search expression, please check again.
 ```
 
+### Viewing a fosterer's detail : `view`
+
+Views a fosterer's details by shifting the foster family list view page to a fosterer's profile view page.
```

Comment:
wait I thought in the profile view page we only only viewing one single fosterer's details HAHA if this is the right idea, maybe it will be better to rephrase to "Views a fosterer's full details" (even though I know right now in the list we are viewing everything, but thats because it's easier to debug when we are able to view everything easily at this stage) to avoid confusion? what do you think!